Code/GraphMol/FileParsers/MolWriters.h

```diff
@@ -210,7 +210,7 @@ class RDKIT_FILEPARSERS_EXPORT TDTWriter : public MolWriter {
   TDTWriter(const std::string &fileName);
   TDTWriter(std::ostream *outStream, bool takeOwnership = false);
 
-  ~TDTWriter();
+  ~TDTWriter() throw();
```

Comment:
It's worth thinking about being systematic about including `throw()` specs, but I think it should be done in a separate PR.
Given that the dtor here ends up calling other methods that are not marked `throw()` this is almost certainly wrong anyway.

Reply:
To be clear, this means remove throw from both interface and implementation?

Reply:
Ok, I’ll do a separate pr.